addons/l10n_dk_audit_trail/__manifest__.py

```diff
@@ -10,6 +10,7 @@
         'l10n_dk',
         'account_audit_trail'
     ],
+    'post_init_hook': '_l10n_dk_audit_trail_post_init',
     'installable': True,
     'auto_install': True,
```

Comment:
btw, what's the use of this module (without your post_init_hook?
Shouldn't it auto_install on `l10n_dk`? (like `l10n_de`) Here you need to install manually the audit trail to have it, no? Don't we want to have it by default for all dbs with l10n_dk installed?

Reply:
When adding the check_audit_trail to true in the company, when installing the module it seems the audit trail options was not set to true even if we did go through the method, which i find weird also. So doing a post init was the only way i found to make it work.
And why this module exist, if i remember correctly it was because we needed to add the account_audit_trail dependency in stable so we created a bridge module, maybe we need to merge those in master though